.github/workflows/deploy_storybook.yml

```
@@ -0,0 +1,38 @@
+name: Test Build and Deploy Storybook to Vercel
+
+on:
+  push:
+    branches:
+      - master
+      - storybook_deploy
+    paths:
+      - packages/components/**
+
+jobs:
+  deploy_storybook:
```

I suggest adding an `if` condition here to restrict the action to the `binary-com/deriv-app` repo so the action doesn't run into errors on people's forks. 

```suggestion
  deploy_storybook:
    if: github.repository == 'binary-com/deriv-app'
```